src/test/scala/edu/ie3/simona/agent/grid/PowerFlowSupportSpec.scala

```diff
@@ -31,12 +35,15 @@ import javax.measure.quantity.Angle
   * order to comprehend the expected test results.
   */
 class PowerFlowSupportSpec
-    extends UnitSpec
+    extends ScalaTestWithActorTestKit
+    with UnitSpec
     with BasicGridWithSwitches
     with PowerFlowSupport
     with GridResultsSupport {
 
-  override val log: LoggingAdapter = NoLogging
+  implicit val log: Logger = LoggerFactory.getLogger(this.getClass)
+  val actorRef: ActorRef[GridAgentMessage] =
+    TestProbe[GridAgentMessage]("noSender").ref
```

Comment:
Maybe use a different name than `noSender`? I think nobody will know where this came from in the future...